skyvern/services/webhook_service.py

```
@@ -163,3 +163,286 @@ def build_sample_workflow_run_payload(run_id: str | None = None) -> str:
 
     payload_dict.update(json.loads(workflow_run_response.model_dump_json(exclude_unset=True)))
     return json.dumps(payload_dict, separators=(",", ":"), ensure_ascii=False)
+from __future__ import annotations
```

Ruff (0.14.1)

`from __future__` imports must occur at the beginning of the file

(F404)